recipe/recipe.yaml

```diff
@@ -1,13 +1,10 @@
 context:
   version: 0.20.1
-  build_number: 5
+  build_number: 6
   # see github.com/conda-forge/conda-forge.github.io/issues/1059 for naming discussion
   # torchvision requires that CUDA major and minor versions match with pytorch
   # https://github.com/pytorch/vision/blob/fa99a5360fbcd1683311d57a76fcc0e7323a4c1e/torchvision/extension.py#L79C1-L85C1
   torch_proc_type: ${{ "cuda" ~ cuda_compiler_version | version_to_buildstring if cuda_compiler_version != "None" else "cpu" }}
-  # Upstream has specific compatability ranges for pytorch and python which are
-  # updated every 0.x release. https://github.com/pytorch/vision#installation
-  compatible_pytorch: 2.5
```

Comment:
was this comment ignored? it isn't entirely safe to leave this "loose" like we do at conda-forge for most pins. I would prefer to keep this comment in, and to use the same old pins.

Reply:
hmm interesting. yeah i somewhat remember flip flopping on these decisions.